warp-dba/src/main/java/com/schibsted/nmp/warp/brands/dba/DbaDarkColors.kt

```diff
@@ -197,6 +198,12 @@ internal object DbaDarkComponentColors : WarpComponentColors {
     override val navBar: WarpNavBarColors = DbaNavBarDarkColors
     override val tooltip: WarpTooltipColors = DbaTooltipDarkColors
     override val switch: WarpSwitchColors = DbaSwitchDarkColors
+    override val spinner: WarpSpinnerColors = DbaSpinnerDarkColors
+}
+
+private object DbaSpinnerDarkColors : WarpSpinnerColors {
```

Comment:
No need to create an object for these as the colors used are not custom component tokens, but already existing semantic color tokens. The spinner colors are defined in https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?gid=1588196525#gid=1588196525